### PR TITLE
Fix legacy imports and update test environments

### PR DESCRIPTION
**Pull request by sarah-witt**
### What does this PR do?

Fixes all instances of legacy imports.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: neutrona/datadog_checks/neutrona/neutrona.py [hunk 1]
```diff
@@ -3,8 +3,8 @@
 import requests
 from requests import RequestException
 
-from datadog_checks.checks import AgentCheck
-from datadog_checks.errors import CheckException
+from datadog_checks.base.checks import AgentCheck
```
Comment (mgarabed) [inline neutrona/datadog_checks/neutrona/neutrona.py:6]:
```suggestion
from datadog_checks.base import AgentCheck
```

File: ping/datadog_checks/ping/ping.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 import platform
 import re
 
-from datadog_checks.checks import AgentCheck
-from datadog_checks.errors import CheckException
-from datadog_checks.utils.subprocess_output import get_subprocess_output
+from datadog_checks.base.checks import AgentCheck
```
Comment (mgarabed) [inline ping/datadog_checks/ping/ping.py:6]:
```suggestion
from datadog_checks.base import AgentCheck
```

File: ping/tests/test_ping.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import mock
 import pytest
 
-from datadog_checks.checks import AgentCheck
-from datadog_checks.errors import CheckException
+from datadog_checks.base.checks import AgentCheck
```
Comment (mgarabed) [inline ping/tests/test_ping.py:4]:
```suggestion
from datadog_checks.base import AgentCheck
```

File: cert_manager/datadog_checks/cert_manager/cert_manager.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # (C) Datadog, Inc. 2019
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-from datadog_checks.checks.openmetrics import OpenMetricsBaseCheck
+from datadog_checks.base.checks.openmetrics import OpenMetricsBaseCheck
```
[the remark this answers is not in the document]
Comment (sarah-witt) [reply cert_manager/datadog_checks/cert_manager/cert_manager.py:4]: Gotcha, I'll make these updates!

File: cert_manager/tests/common.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # (C) Datadog, Inc. 2019
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-from datadog_checks.stubs import aggregator
+from datadog_checks.base.stubs import aggregator
```
Comment (ofek) [inline cert_manager/tests/common.py:4]:
```suggestion
```

File: logstash/tests/common.py [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import os
 
-from datadog_checks.utils.common import get_docker_hostname
+from datadog_checks.base.utils.common import get_docker_hostname
```
Comment (ofek) [inline logstash/tests/common.py:3]:
```suggestion
from datadog_checks.dev import get_docker_hostname
```

File: riak_repl/tests/common.py [hunk 7]
```diff
@@ -1,6 +1,6 @@
 import os
 
-from datadog_checks.utils.common import get_docker_hostname
+from datadog_checks.base.utils.common import get_docker_hostname
```
Comment (ofek) [inline riak_repl/tests/common.py:3]:
```suggestion
from datadog_checks.dev import get_docker_hostname
```

File: sortdb/tests/test_sortdb.py [hunk 8]
```diff
@@ -20,7 +20,7 @@ def instance():
 
 @pytest.fixture()
 def aggregator():
```
Comment (ofek) [inline sortdb/tests/test_sortdb.py:22]: Can remove

File: cert_manager/tests/common.py (hunk 5 above)
Comment (sarah-witt) [reply cert_manager/tests/common.py:4]: `aggregator` is used in this file

Comment (ofek) [reply cert_manager/tests/common.py:4]: https://datadoghq.dev/integrations-core/ddev/plugins/#fixtures

**Pull request merged**